### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by LeeBergstrand**

### CODE REVIEW

File: Visualization/ExtractBackBlastSubject.py [hunk 1]
```diff
@@ -1,4 +1,6 @@
-#!/usr/bin/env python 
+#!/usr/bin/env python
```
Comment (jmtsuji) [inline Visualization/ExtractBackBlastSubject.py]: @LeeBergstrand This is not included in your draw.io diagram. What does it do?

File: Visualization/ExtractBackBlastSubject.sh [hunk 2]
```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
Comment (jmtsuji) [inline Visualization/ExtractBackBlastSubject.sh]: @LeeBergstrand This is not included in your draw.io diagram. What does it do?

File: Visualization/ExtractBackBlastSubject.sh [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 # A simple script that extracts BLAST hit subjects for a certain set of query accessions from from BackBlast results.
 
 for file;
 do
-	echo Etracting subjects from $file
-	python /Users/lee/Dropbox/RandD/Repositories/BackBLAST-Gene-Cluster-Finder/Visualization/HeatMap/ExtractBackBlastSubject.py $file /Users/lee/Dropbox/RandD/Repositories/BackBLAST-Gene-Cluster-Finder/Visualization/HeatMap/TestData/SecondRun/CompleteResults.csv
+	echo Etracting subjects from ${file}
+	python /Users/lee/Dropbox/RandD/Repositories/BackBLAST-Gene-Cluster-Finder/Visualization/HeatMap/ExtractBackBlastSubject.py ${file} /Users/lee/Dropbox/RandD/Repositories/BackBLAST-Gene-Cluster-Finder/Visualization/HeatMap/TestData/SecondRun/CompleteResults.csv
```
Comment (jmtsuji) [inline Visualization/ExtractBackBlastSubject.sh]: Would need to generalize the script path.

File: SupplementaryScripts/runAABlast.sh [hunk 4]
```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (LeeBergstrand) [reply SupplementaryScripts/runAABlast.sh]: For running one-way BLAST manually. It's not used.

File: SupplementaryScripts/makeAABlastDB.sh [hunk 5]
```diff
@@ -1,10 +1,10 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (LeeBergstrand) [reply SupplementaryScripts/makeAABlastDB.sh]: Yes. I agree.

File: SupplementaryScripts/runBackBlast.sh [hunk 6]
```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/usr/bin/env bash
```
[the remark this answers is not in the document]
Comment (LeeBergstrand) [reply SupplementaryScripts/runBackBlast.sh]: Yes

File: SupplementaryScripts/runAABlast.sh (hunk 4 above)
[the remark this answers is not in the document]
Comment (jmtsuji) [reply SupplementaryScripts/runAABlast.sh]: @LeeBergstrand Can it be deleted, then? (Or moved to a separate repo?)

File: Visualization/ExtractBackBlastSubject.py (hunk 1 above)
Comment (jmtsuji) [reply Visualization/ExtractBackBlastSubject.py]: Can they be deleted or moved to a separate repo?

**Pull request merged**